### PR TITLE
Implemented Lerna

### PR DESCRIPTION
**Pull request by szul**
This should set Lerna up correctly. It does not include publishing.

### CODE REVIEW

File: libraries/botbuilder-config/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "./lib/index.js",
   "typings": "./lib/index.d.ts",
   "scripts": {
-    "postinstall": "./node_modules/.bin/tsc -p .",
+    "postinstall": "tsc",
```
Comment (iMicknl) [inline libraries/botbuilder-config/package.json]: Why did you change this to `tsc` only? Having the previous version with the full node_modules path means that it doesn't have to be globally installed and that it will use the Typescript version defined in the dependencies. Or is my idea not correct? :)

(I guess it will be replace by something like `lerna build` eventually.

Comment (szul) [reply libraries/botbuilder-config/package.json]: @iMicknl No, you’re right. Should file this as an issue. The lerna build command was throwing errors about “.” not being a command, and reverting to just tsc resolved it, but I’d prefer getting it working the other way.
